Working Branch

Amita-Roy opened the pull request:
Hi, we have cloned this project from [https://www.newsweek.com/](url). The main focus for this project was to work with Bootstrap.

- We have added Bootstrap as a npm package
- To work with bootstrap variables we are using main.scss file which is later compiling to style.css.
- Our customised style are in **SCSS/main.scss** file and we have corrected all the stylelint error for our main.scss file.
- Html file is also validated

Amita-Roy commented:
we have made all the changes required in the review and we have added Bootstrap as per the link provided in the odin project. Please refer the screenshot.

![bootstrap](https://user-images.githubusercontent.com/47393794/87223762-2702d400-c39d-11ea-94da-cce2cdca5e92.png)

Amita-Roy commented:
Thank you for the reviews, we have made all the required changes.

bertil291utn commented:
**Status:** Changes requested 🔴  :shipit:? Not yet.

# Hey guys, you've been doing a great job, and also I hope you enjoyed coding.


You have achieved an amazing job with your README file, commit messages and your GitHub Flow, however in order to accomplish as the actual page, you required to do these changes, listed below: 

**Small screen**

- [ ] **Footer section.-** Copyright section should be under Newsweek logo 

| ACTUAL WEB PAGE                                              | YOURS                                                        |
| ------------------------------------------------------------ | ------------------------------------------------------------ |
| ![image](https://user-images.githubusercontent.com/24902525/87233069-d930a080-c389-11ea-93f8-1e990280e7b8.png) | ![image](https://user-images.githubusercontent.com/24902525/87233067-ce760b80-c389-11ea-9815-af94be5cf21d.png) |

- [ ] **Category tag .-** Each article has a tag call category, try to adjust each element width on each article on all sections.

| ACTUAL WEB PAGE                                              | YOURS                                                        |
| ------------------------------------------------------------ | ------------------------------------------------------------ |
| ![image](https://user-images.githubusercontent.com/24902525/87233129-55c37f00-c38a-11ea-848a-5af772d97667.png) | ![image](https://user-images.githubusercontent.com/24902525/87233111-2b71c180-c38a-11ea-96d3-bd2e635922d1.png) |

- [ ] **Category tag .-** Each article has a tag call category, **IN THE MAGAZINE** section, try to work with a red background and red color font

| ACTUAL WEB PAGE                                              | YOURS                                                        |
| ------------------------------------------------------------ | ------------------------------------------------------------ |
| ![image](https://user-images.githubusercontent.com/24902525/87233300-b69f8700-c38b-11ea-8ced-5e27d2ebd96f.png) | ![image](https://user-images.githubusercontent.com/24902525/87233265-7b9d5380-c38b-11ea-98c9-09ba4802c2db.png) |

**Medium screen**

- [ ] Reduce the size of each avatar on the **THE DEBATE** section

<img src="https://user-images.githubusercontent.com/24902525/87233369-46ddcc00-c38c-11ea-927c-091620a041e1.png" alt="image" style="zoom:67%;" />

- [ ] Use `object-fit: cover;` to avoid stretching images on **IN THE MAGAZINE SECTION**. Same for a large screen in this section

<img src="https://user-images.githubusercontent.com/24902525/87233436-18acbc00-c38d-11ea-8b8f-19cb124c429f.png" alt="image" style="zoom:67%;" />

- [ ]  After the editor's pick section, try to fill the blue square should be the same height as your text. Remember to work on each section where you have a similar situation. Same for a large screen in  this section

<img src="https://user-images.githubusercontent.com/24902525/87233477-51e52c00-c38d-11ea-932f-b9cd1d7d72dc.png" alt="image" style="zoom:67%;" />


Guys work in the requested changes above and request another review.
Good luck. 
<hr>

> “Never stop, never conform until what is good becomes even better, and what’s better becomes excellent.”

eypsrcnuygr commented:
Copyright section under the logo for the.
tags are adjusted.
In the magazine section background and color changed.
The debate section images reduced.
In the magazine section, images don't stretch now.
Editors pick section, images having the full height except for the first ones.

eypsrcnuygr commented:
In the magazine section, the images are bigger for the xl screens.

Amita-Roy commented on styles/styles.css:
```diff
@@ -0,0 +1,7409 @@
+  vertical-align: middle !important; }
+
+.align-bottom {
+  vertical-align: bottom !important; }
+
+.align-text-bottom {
+  vertical-align: text-bottom !important; }
+
+.align-text-top {
+  vertical-align: text-top !important; }
+
+.bg-primary {
+  background-color: red !important; }
```
Our customised styles are in main.scss. The styles.css is compiling from bootstrap which is nothing to do with our customised styles.

Amita-Roy commented:
We removed all the inline style from html.
Removed all the unused classes from html.

Our customised styles are in main.scss. The styles.css is compiling from bootstrap which is nothing to do with our customised styles.

kitpao commented:
# 👋 Hello Team,

Nice work so far! 👏 You are almost done! 

🔍You only missed these details: 

- [ ] Your Subscribe and Newsletter titles are still in black instead of red

- [ ] Please, position this image correctly in the mobile version (centered)

Do this and you will be done!

![2020-07-13](https://user-images.githubusercontent.com/53256139/87316352-4871d600-c4eb-11ea-99c3-4b0c12d4a6bb.png)
___
##### 👉 If you have any doubts you can contact me in slack as @kitpao

🔗 __[Github](https://github.com/kitpao) | [Twitter](https://twitter.com/Kitpao1) | [LinkedIn](https://www.linkedin.com/in/kitzia-paola-vidal/) | [Medium](https://medium.com/@kitpao91)__
